nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/flow-designer/state/flow/flow.effects.ts

```diff
@@ -2495,45 +2494,6 @@ export class FlowEffects {
         )
     );
 
-    copy$ = createEffect(() =>
```

Comment:
I think we can also remove the `copy` action since we're not removing the effect for it.